impl/ecl/src/main/resources/org/wildfly/extras/transformer/eclipse/jakarta-direct.properties

```diff
@@ -139,6 +139,10 @@ javax.servlet.jsp.jstl.sql.userName=jakarta.servlet.jsp.jstl.sql.userName
 # Jakarta Server Faces
 javax.faces.validator.beanValidator.ValidatorFactory=jakarta.faces.validator.beanValidator.ValidatorFactory
 
+# Jakarta Java API for XML Binding
+javax.xml.bind.JAXBContext=jakarta.xml.bind.JAXBContext
+javax.xml.bind.JAXBContextFactory=jakarta.xml.bind.JAXBContextFactory
```

Comment:
@ropalka I think the deprecated JAXB 1.0 prop needs to be handled as well:

https://github.com/eclipse-ee4j/jaxb-api/blob/master/jaxb-api/src/main/java/jakarta/xml/bind/ContextFinder.java#L42

javax.xml.bind.context.factory=jakarta.xml.bind.context.factory

Reply:
Thanks @bstansberry - I didn't notice that property.